### ISSUE TITLE
Maintenance status

### ISSUE THREAD

**Issue by Veetaha** [softprops/dynomite#144]
Looks like this project requires more maintainership to keep up.
@softprops, seems like your availability cadence is not enough, what do you think about expanding the maintainers list?

**Comment by softprops** [comment]
That's def something I'd appreciate. I'm juggling busy work a new baby and a bunch of oss projects

**Comment by softprops** [comment]
Also heads up, I just realized I haven't been getting GitHub notif emails for a while. I'm trying to resolve that now. That's another source of delayed response for me. I'm not getting any email from GitHub atm!

**Comment by rimutaka** [comment]
I'm happy to chip in with code reviews. Just tag me if you need an extra pair of eyes.

**Comment by ZaMaZaN4iK** [comment]
For anyone, who is interested in maintaining this library and is ready to do the required work, feel free to do it here: https://github.com/rust-serverless/dynomite

**Comment by Veetaha** [comment]
@softprops Let's invite @ZaMaZaN4iK to the owners of `dynomite` crate on `crates.io` so we can publish the new version it (probably using CI with github secrets to do the job)?

**Comment by Veetaha** [comment]
Ping, @softprops, we can't continue improving the crate if we can't make new releases...
E.g. this change won't get released: https://github.com/rust-serverless/dynomite/pull/4

**Comment by softprops** [comment]
Hi folks there must have mean a mis-communication somewhere. I don't recall agreeing to moving the development for this crate to another repo? 

Forks are welcome but this is the canonical repository

**Comment by softprops** [comment]
> For anyone, who is interested in maintaining this library and is ready to do the required work, feel free to do it here: https://github.com/rust-serverless/dynomite

I do not appreciate this kind of thing. Declaring one's self a canonical replacement without consent if an author is closed to boarder line steeling.  I've only consented to expanding the list of maintainers for those that would like to volunteer above

**Comment by ZaMaZaN4iK** [comment]
Yeah, this was a kinda misunderstanding here, I agree. Sorry for that from my side.

**Comment by Veetaha** [comment]
@softprops, I would like to volunteer for `dynomite`. We use it in our company extensively and miss some features. I've already submitted a PR to https://github.com/rust-serverless/dynomite/pull/4 for one feature that we miss and I'll try to keep the updates going on. I also contributed to this crate in the past (https://github.com/softprops/dynomite/pull/134, https://github.com/softprops/dynomite/pull/132) and am very familiar with the codebase.

I see that you have low capacity so I want to ask you if you could share with me all or any of the permissions:
- To merge PRs into this same repository (softprops/dynomite)
- To invoke the `publish-crate` workflow, which requires making new git tags on the repo
- Invite me to the owners of the crate on `crates.io` (though this is not strictly required if i just can run the publish-crate workflow here)
